### ISSUE TITLE
Debian 11 Packagecloud repository is empty

### ISSUE THREAD

**Issue by nodiscc** [netdata/netdata#11620]
##### Bug report summary

After following the instructions on https://packagecloud.io/netdata/netdata/install on Debian 11 stable (Bullseye), no `netdata` package is available from packagecloud repositories:

```bash
$ cat /etc/apt/sources.list.d/netdata_netdata.list
deb https://packagecloud.io/netdata/netdata/debian/ bullseye main

$ apt policy netdata
netdata:
  Installed: 1.29.3-4
  Candidate: 1.29.3-4
  Version table:
 *** 1.29.3-4 500
        500 http://deb.debian.org/debian bullseye/main amd64 Packages
        500 http://deb.debian.org/debian bullseye/main i386 Packages
        100 /var/lib/dpkg/status
```

##### OS / Environment

```
Debian 11.0.0
```

##### Netdata version

None

##### Installation method

https://packagecloud.io/netdata/netdata/install

##### Component Name

Packaging

##### Steps To Reproduce

Follow the instructions at https://packagecloud.io/netdata/netdata/install on a standard Debian Stable system

##### Expected behavior

The `netdata` package from packagecloud repositories should appear in the available/installable versions of the package.

Related: https://github.com/netdata/netdata/issues/8882

See https://packagecloud.io/netdata/netdata/debian/dists/bullseye/Contents-amd64.gz

**Comment by Ferroin** [comment]
> Is it not possible to trigger a build for the last stable release?

Not without it republishing _everything_ else at the same time.

An overhaul of our release engineering process is the next big project on the list for @netdata/agent-sre once the current project of overhauling the installer is done, and dealing with this particular issue is near the top of the list of things to fix.

**Comment by nodiscc** [comment]
This was fixed with the release of [netdata v1.32.0](https://github.com/netdata/netdata/releases/tag/v1.32.0)

```bash
$ apt-cache policy netdata
netdata:
  Installed: 1.29.3-4
  Candidate: 1.32.0
  Version table:
     1.32.0 500
        500 https://packagecloud.io/netdata/netdata/debian bullseye/main amd64 Packages
 *** 1.29.3-4 500
        500 http://deb.debian.org/debian bullseye/main amd64 Packages
        500 http://deb.debian.org/debian bullseye/main i386 Packages
        100 /var/lib/dpkg/status
```

The debian 11 package is affected by another bug which I will report in a separate issue.
